metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -1018,28 +1018,31 @@ def _process_table(
     def get_table_properties(
         self, inspector: Inspector, schema: str, table: str
     ) -> Tuple[Optional[str], Optional[Dict[str, str]], Optional[str]]:
+        location: Optional[str] = None
+        description: Optional[str] = None
+        properties: Dict[str, str] = {}
+
         try:
             location: Optional[str] = None
             # SQLALchemy stubs are incomplete and missing this method.
             # PR: https://github.com/dropbox/sqlalchemy-stubs/pull/223.
             table_info: dict = inspector.get_table_comment(table, schema)  # type: ignore
         except NotImplementedError:
-            description: Optional[str] = None
-            properties: Dict[str, str] = {}
+            return description, properties, location
         except ProgrammingError as pe:
             # Snowflake needs schema names quoted when fetching table comments.
             logger.debug(
                 f"Encountered ProgrammingError. Retrying with quoted schema name for schema {schema} and table {table}",
                 pe,
             )
-            description = None
-            properties = {}
             table_info: dict = inspector.get_table_comment(table, f'"{schema}"')  # type: ignore
-        else:
-            description = table_info["text"]
 
-            # The "properties" field is a non-standard addition to SQLAlchemy's interface.
-            properties = table_info.get("properties", {})
+        description = table_info["text"]
```

Comment:
Is it possible that `table_info` doesn't have a `text` property?
Maybe `table_info.get("text")` would be safer.